pkg/cluster/deploystorage.go

```diff
@@ -191,6 +186,144 @@ func (m *manager) deployStorageTemplate(ctx context.Context, installConfig *inst
 			m.clusterNSG(infraID, installConfig.Config.Azure.Region),
 		},
 	}
+	if feature.IsRegisteredForFeature(m.subscriptionDoc.Subscription.Properties, "Microsoft.RedHatOpenShift/RedHatEngineering") {
+		//RedHatEngineering feature flag
+		roles := []*arm.Resource{
+			rbac.ResourceGroupRoleAssignmentWithName(
+				rbac.RoleReader,
+				"'"+clusterSPObjectID+"'",
+				"guid(resourceGroup().id, 'SP / RG Reader')", false),
+			rbac.ResourceRoleDefinitionWithName("guid(resourceGroup().id, 'SP / RG CSP')",
```

Comment:
Comment that `/read` actions are covered via ReadersRole would be good :)

Reply:
I changed the name to "Azure Red Hat OpenShift autogenerated RG CSP", points out where the role definition and assignment came from, and that it's autogenerated.